NuGet.config

```diff
@@ -22,6 +22,8 @@
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
     <!-- Used for BenchmarkDotNet prerelease packages -->
     <add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
+    <!-- Used for Roslyn version 4.5.0-2.23107.1 -->
+    <add key="general-testing" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing/nuget/v3/index.json" />
```

Comment:
Is there a reason we couldn't get core-eng to put these packages on the standard feed?

Reply:
I manually published this before https://github.com/dotnet/roslyn/pull/66570 was merged. I chose General Testing feed because I wasn't sure where I'm allowed to push and not break things. But it's not necessary anymore, since https://github.com/dotnet/roslyn/pull/66570 was merged, so I'm able to use the official version.